index.html

```diff
@@ -126,6 +126,9 @@
 								<li class="routing">
 									<a href="examples/troopjs_require/" data-source="https://github.com/troopjs/" data-content="TroopJS attempts to package popular front-end technologies and bind them with minimal effort for the developer. It includes jQuery for DOM manipulation, When.js for promises, RequireJS for modularity and Has.js for feature detection. On top, it includes Pub/Sub support, templating, weaving (widgets to DOM) and auto-wiring.">TroopJS + RequireJS</a>
 								</li>
+								<li class="routing">
+									<a href="examples/openui5/" data-source="https://github.com/sap/openui5" data-content="OpenUI5 lets you build enterprise-ready web applications, responsive to all devices, running on almost any browser of your choice.">OpenUI5</a>
+								</li>
```

Comment:
Don't directly put the app here, all apps start in labs first :) We can discuss putting it in the "main" section after it's been test running online for a while and when we we're sure it's gonna stay maintained.